ui/packages/shared/profile/src/ProfileSource.tsx

```diff
@@ -56,6 +56,10 @@ export function SuffixParams(params: {[key: string]: any}, suffix: string): {[ke
 }
 
 export function ParseLabels(labels: string[]): Label[] {
+  if (labels.length === 0 || (labels.length === 1 && labels[0] === '')) {
```

Comment:
instead of this, you could also strip the labels array of any empty string values and then check its length:

```
const nonEmptyLabels = labels.filter((str) => str !== '');
if (labels.length < 1)
```

Reply:
Like that much better! :+1: